### PR TITLE
Update governance rules for GrimoireLab project

### PR DESCRIPTION
**Pull request by sduenas**
Governance rules have been updated to make them more clear and to align the project with CHAOSS. This commit also improves the rules and recommendations for contributing to the project.

### CODE REVIEW

**Comment by vchrombie** [comment]
Resolves #592

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# GrimoireLab Maintainers
+
+- Santiago Dueñas (@sduenasd) - Tech Lead & Developer
```
Comment (vchrombie) [inline MAINTAINERS.md:3]:
```suggestion
- Santiago Dueñas (@sduenas) - Tech Lead & Developer
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,218 +1,88 @@
...
-Committers may resign if they are no longer involved enough in the
-project. Committers can also propose the removal of other committer
-rights, in case they are no longer involved in the project. For being
-removed, after the proposal there will be an action voting among
-committers, except for the one being proposed to be removed.
-
-The current list of committers is (GitHub handles):
+If you have ideas for improvements or new features, feel free to open a new
+discussion or issue to share them. We appreciate well-documented
+enhancement suggestions. You can also have at look to our [ROADMAP](./ROADMAP.md)
```
Comment (jjmerchante) [inline CONTRIBUTING.md:37]: The ROADMAP.md file does not exist. Is it something to add in the future?

**Review by jjmerchante — COMMENTED** [review]
LGTM just a single comment

File: CONTRIBUTING.md (hunk 2 above)
Comment (sduenas) [reply CONTRIBUTING.md:37]: Yes, I have it ready and I will upload it later today after this PR is merged.

**Review by zhquan — APPROVED** [review]
LGTM

**Pull request merged**